README.md

```diff
@@ -158,9 +159,47 @@ You can use the SharedPreferenceVault with SecretKey generated entirely from the
         SharedPreferenceVaultRegistry.getInstance().getVault(VAULT_ID).rekeyStorage(secretKey);
 
 #### Rekeying
-The vault can be rekeyed at any time. This will delete all values in the shared 
-preference file. This is completely irreversible.
+The vault can be rekeyed at any time. This will delete all values in the shared preference file. This is completely irreversible.
+
+#### API Changes to Commit and Apply
+Typical results of the SharedPreferenc commit and apply are slightly modified in Vault resulting of the extra encryption that is taking place. Because of this, commit and apply can both fail for reasons other than the SharedPreference write failing. The return type for commit is now more important. The recommendation is to **enable exceptions** when creating a SharedPreferenceVault.
```

Comment:
@lazysp333 Suggested changes:
Typical behavior of the SharedPreference commit and apply methods are slightly modified in Vault as a result of the added encryption. Because of this, commit and apply can both fail for reasons other than the SharedPreference write failing. This means that you should pay attention to the return value for commit and react appropriately in the case where it returns false. It is recommended to **enable exceptions** when creating a SharedPreferenceVault if you want to see more information on failures (by catching the thrown exception yourself)